github_traffic_stats.py

```diff
@@ -26,10 +27,13 @@ def collect(db, user, passwd, token, repo, org):
             sys.exit('Username "' + user + '" does not have collaborator permissions in repo "' + repo + '"')
     views_14_days = gh.repos(org, repo).traffic.views.get()
     found_new_data = False
+    time_format = "%Y-%m-%d"
     for view_per_day in views_14_days['views']:
         timestamp = view_per_day['timestamp']
+        timestamp = datetime.datetime.strptime(timestamp,"%Y-%m-%dT%H:%M:%SZ")
+        timestamp = timestamp.strftime(time_format)
```

Comment:
I just noticed you're also changing the date format. This will make the new version non backward compatible to the older version which might be an issue (at least for me 😃). So I see 2 options here:

- Revert the date format change
- Add a command line parameter to define the date format. The default should be the one in the current version

Reply:
Hi, 
So, I need a clarification. When you say revert the date format change, in this scenerio, 
1. Is it acceptable to convert the python unicode object ([timestamp] which caused the issue to begin with) into  python timestamp object AND keep the format same as you had?

Reply:
Yes, this is correct. Let's just make sure the date time format stays the same

Reply:
completed the request. You can review.
1. reverted back the changes on README. It now is same as your master README
2. Changed the timestamp variable from unicode object to string so that it could be used as a key in pickedb while keep the format as requested.